### PR TITLE
implemented basic functionality

### PR DESCRIPTION
**Pull request by VSemenovykh**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -13,27 +13,31 @@
...
+    </dependency>
+    <dependency>
+        <groupId>org.projectlombok</groupId>
+        <artifactId>lombok</artifactId>
+        <optional>true</optional>
+    </dependency>
+    <dependency>
+         <groupId>javax.xml.bind</groupId>
+         <artifactId>jaxb-api</artifactId>
+    </dependency>
```
Comment (AndrewSmrnv) [inline pom.xml:41]: Please format the pom file

File: src/main/java/ru/ncedu/repository/AutoRepository.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+import ru.ncedu.entity.Auto;
+
+import java.util.List;
+
+@Repository	
+public interface AutoRepository extends PagingAndSortingRepository<Auto, Long>, JpaSpecificationExecutor<Auto> {
```
Comment (AndrewSmrnv) [inline src/main/java/ru/ncedu/repository/AutoRepository.java:12]: Do we need JpaSpecificationExecutor?

File: src/main/java/ru/ncedu/service/BrandServiceImp.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    BrandRepository brandRepository;
+
+    @Override
+    public List<Brand> getAllBrand(){
+       return brandRepository.findAll();
+    }
+
+    @Override
+    public List<Brand> findAll(int pageNumber, int rowPerPage) {
+        List<Brand> contacts = new ArrayList<>();
```
Comment (AndrewSmrnv) [inline src/main/java/ru/ncedu/service/BrandServiceImp.java:29]: Please rename the variable

File: pom.xml [hunk 4]
```diff
@@ -32,8 +36,8 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <groupId>javax.xml.bind</groupId>
```
Comment (VSemenovykh) [inline pom.xml:39]: refactored

**Review by VSemenovykh — COMMENTED** [review]
deleted interface the JpaSpecificationExecutor

File: src/main/java/ru/ncedu/repository/AutoRepository.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package ru.ncedu.repository;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+import ru.ncedu.entity.Auto;
+
+import java.util.List;
+
+@Repository
+public interface AutoRepository extends PagingAndSortingRepository<Auto, Long> {
```
Comment (VSemenovykh) [inline src/main/java/ru/ncedu/repository/AutoRepository.java:10]: deleted interface the JpaSpecificationExecutor

File: pom.xml (hunk 1 above)
Comment (VSemenovykh) [reply pom.xml]: done

File: src/main/java/ru/ncedu/service/BrandServiceImp.java (hunk 3 above)
Comment (VSemenovykh) [reply src/main/java/ru/ncedu/service/BrandServiceImp.java]: done

**Pull request merged**